dashboard/app/models/user.rb

```diff
@@ -770,6 +772,10 @@ def needs_to_backfill_user_scripts?
       !user_levels.empty?
   end
 
+  def account_age_days
+    DateTime.now - created_at.to_datetime
```

Comment:
This (`DateTime.now - created_at.to_datetime`) is an object of type `Rational`. Perhaps cast it to an integer, document the method returns an integer,  and comment that the method rounds down?